mumbleBot.py

```diff
@@ -71,7 +71,25 @@ def __init__(self, args):
             tt.daemon = True
             tt.start()
 
-        self.mumble = pymumble.Mumble(args.host, user=args.user, port=args.port, password=args.password,
+        if args.host:
+            host = args.host
+        else:
+            host = var.config.get("server", "host")
+        if args.port:
+            port = args.port
+        else:
+            port = var.config.getint("server", "port")
```

Comment:
You need a default at 64738 if no port is specified.

Reply:
configuration.default.ini includes 64738 as the default

Reply:
good point !